src/tests/work-item/work-item-list/work-item-detail.page.js

```diff
@@ -289,17 +291,19 @@ class WorkItemDetailPage {
   }
 
   /* The user assigned to the workitem */
-  get workItemDetailAssignee () {
+  workItemDetailAssignee () {
     return element(by.xpath(".//*[contains(@class,'detail-assignee-name')]"));
   }
-
+  details_assigned_user () {
+    return element(by.id("WI_details_assigned_user"));
+  }
   get clickWorkItemDetailAssignee () {
     return this.workItemDetailAssignee.click();
   }
 
   /* Search string box for the user to assign to the workitem */
   get workItemDetailAssigneeSearch () {
-    return element(by.css(".list-container>input"));
+    return element(by.xpath('.//*[@id="userAssigneeSearchInput"]'));
```

Comment:
I think that we can replace this line with:
`
return element(by.id('userAssigneeSearchInput'));`

Sometime son - next week? - we should try to remove all the other xpath queries from the tests as they are brittle and difficult to maintain.